test/system/standalone/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+# test compile and execution of a cupla program using stand allown header
+# $1 C++ compiler name
+
+test_code_dir=$(dirname $0)
+
+# compiles the test case for the given accelerator
+# $1 flag if the compiled binary should be executed, 1 == execute binary else do not test binary 
+# $2 compiler name
+# $3 accelerator name (CpuOmp2Blocks, CpuOmp2Threads, CpuSerial, CpuThreads, GpuCudaRt, GpuHipRt)
+# $4 additional compiler flags
+function compile {
+    execute_bin=$1
+    compiler_name=$2
+    acc_name=$3
+    if [ $# -eq 4 ] ; then
+        compiler_flags="$4"
+    fi
+    ret=$($compiler_name $test_code_dir/main.cpp $test_code_dir/kernel.cpp  -I$test_code_dir/../../../include -I$test_code_dir/../../../alpaka/include -std=c++11 -DCUPLA_ACC_$acc_name -DCUPLA_HEADER_ONLY -o $acc_name $compiler_flags && \
```

Comment:
Use `${varname}` for variables to allow seamless combination with strings (and better bash style) [[1](https://github.com/azet/community_bash_style_guide)]:
```suggestion
    ret=$(${compiler_name} ${test_code_dir}/main.cpp ${test_code_dir}/kernel.cpp  -I${test_code_dir}/../../../include -I${test_code_dir}/../../../alpaka/include -std=c++11 -DCUPLA_ACC_${acc_name} -DCUPLA_HEADER_ONLY -o ${acc_name} ${compiler_flags} && \
```